app/package.json

```diff
@@ -2,39 +2,26 @@
   "name": "scalarp-kiosk",
   "version": "1.0.0",
   "description": "A kiosk app to manage LARPs",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
-  "license": "MIT",
-  "devDependencies": {
-    "babel-preset-es2015": "~6.24.1",
-    "babel-preset-react": "~6.24.1",
-    "babelify": "~7.3.0",
-    "bootstrap": "~3.3.7",
-    "browserify": "~14.3.0",
-    "chai": "^3.5.0",
-    "del": "~2.2.2",
-    "eslint": "~3.19.0",
-    "eslint-config-airbnb": "~14.1.0",
-    "eslint-plugin-import": "~2.2.0",
-    "eslint-plugin-jsx-a11y": "~4.0.0",
-    "eslint-plugin-react": "~6.10.3",
-    "gulp": "~3.9.1",
-    "gulp-bootstrap-configurator": "~0.4.1",
-    "gulp-eslint": "~3.0.1",
-    "gulp-mocha": "^4.3.1",
-    "mocha": "^3.3.0",
-    "vinyl-source-stream": "~1.1.0"
-  },
+  "private": true,
   "dependencies": {
-    "prop-types": "~15.5.8",
-    "react": "~15.5.4",
-    "react-bootstrap": "~0.31.0",
-    "react-dom": "~15.5.4",
+    "bootstrap": "^4.0.0-alpha.6",
+    "react": "^15.6.1",
+    "react-addons-css-transition-group": "^15.6.0",
+    "react-addons-transition-group": "^15.6.0",
+    "react-dom": "^15.6.1",
+    "react-icons": "^2.2.5",
     "react-redux": "^5.0.5",
+    "reactstrap": "^4.8.0",
```

Comment:
Looks like the switch also removed some CSS padding / margin.
If you look at before/after, the UI renders boxes without any.